pages/exam/index.tsx

```diff
@@ -68,6 +68,12 @@ const Exam: NextPage = () => {
     }
   }, [checkPassed, remainingTime]);
 
+  const totalSeconds = 3 * 60;
```

Comment:
why is this hardcoded?

Reply:
if you want i can change this. for something like this 
const totalTimeInSeconds = minutes * 60 + seconds;

Reply:
the time of the exam may vary, can be 15 minutes or less, I put 3 minutes to test it

Reply:
i changed it and made another pull request it is not fixed anymore

Reply:
why did you made another PR?

Reply:
you dont want it hardcoded so i changed it

Reply:
oh, you made another commit, you can set the pr ready for review

Reply:
yeah sorry for confusion